### ISSUE TITLE
Add Ext JS 4.2.2 to the Deft JS Test Suite

### ISSUE THREAD

**Issue by johnyanarella** [deftjs/DeftJS#118]

**Comment by bl1nder** [comment]
Deft JS can work with Ext 4.2.1 now?

**Comment by johnyanarella** [comment]
Yes, the Deft JS v0.9 test suite passes for all current versions of Ext JS, including 4.2.1 and 4.2.2.

We're just working on making these results available via the public test suite as part of our continuous integration process.  (4.2.2 was recently removed from the CDN because it is a support release, so this poses some difficulty toward meeting that goal.)
